examples/applications/plot_topics_extraction_with_nmf_lda.py

```diff
@@ -1,8 +1,7 @@
 """
-========================================================
-Topic extraction with Non-negative Matrix Factorization\
-and Latent Dirichlet Allocation
-========================================================
+=======================================================================================
```

Comment:
Oh, maybe it does violate PEP8. I hardly realised this was in an example.

I'm okay to see NMF here if you wish. I'm also okay with the pep8 eror